### PR TITLE
Updates to LO Cards

### PR DESCRIPTION
**Pull request by katieblomquist**
Updated the LO cards based on mocks. 
Also updated pagination to new branded styling. 

https://www.figma.com/file/5SXdtpGPxjHUZWdTIYEkaZ/Learning-Object-Card-Redeesign?node-id=0%3A1&t=nHdo8hn3YKp0Djoo-1

Screen Shot 2023-01-26 at 8 54 13 AM
Screen Shot 2023-01-26 at 8 54 24 AM
Screen Shot 2023-01-26 at 8 54 40 AM
Screen Shot 2023-01-26 at 8 54 57 AM
Screen Shot 2023-01-26 at 2 02 47 PM

### CODE REVIEW

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by mfranz1 — APPROVED** [review]

**Comment by mfranz1** [comment]
Really good job on this 🙌

**Review by pzalep1 — APPROVED** [review]
Goodbye to the green boxes! Looks awesome!

File: src/app/cube/shared/learning-object/learning-object.component.ts [hunk 1]
```diff
@@ -130,16 +132,29 @@ export class LearningObjectListingComponent implements OnInit, OnChanges, OnDest
...
-      } else if (organization) {
-        return titleCase(organization);
-      }
+  organizationFormat(organization: string) {
+    if (organization && (organization.charAt(1) === organization.charAt(1).toUpperCase())) {
+      return organization;
+    } else if (organization) {
+      return titleCase(organization);
     }
+  }
```
Comment (pmanal1) [inline src/app/cube/shared/learning-object/learning-object.component.ts:141]:
```suggestion
  organizationFormat(organization: string) {
    return titleCase(organization);
  }
```
Does this do the same thing?

Comment (katieblomquist) [reply src/app/cube/shared/learning-object/learning-object.component.ts]: One was checking if it was in all caps and then just returning it. and then title casing everything else, but it does work as a one-liner. I went ahead and changed it.

**Review by pmanal1 — APPROVED** [review]
🎨👩‍🎨 Beautiful

**Pull request merged**